app/assets/javascripts/controllers/topology/container_topology_controller.js

```diff
@@ -218,20 +218,20 @@ function ContainerTopologyCtrl($scope, $http, $interval, topologyService, $windo
     $scope.relations = data.data.relations;
     $scope.kinds = data.data.kinds;
     icons = data.data.icons;
-    var size_limit = data.data.settings.containers_max_items;
 
     if (currentSelectedKinds && (Object.keys(currentSelectedKinds).length !== Object.keys($scope.kinds).length)) {
       $scope.kinds = currentSelectedKinds;
-    } else if (size_limit > 0) {
+    } else if (data.data.settings && data.data.settings.containers_max_items) {
+      var size_limit = data.data.settings.containers_max_items;
       var remove_hierarchy = ['Container',
-        'ContainerGroup',
-        'ContainerReplicator',
-        'ContainerService',
-        'ContainerRoute',
-        'Host',
-        'Vm',
-        'ContainerNode',
-        'ContainerManager'];
+                              'ContainerGroup',
```

Comment:
Note this is the wrong align for JS arrays. It was right before (except for the ending `]` which should be on it's own line).